fractal/client/cmd/_task.py

```diff
@@ -14,14 +17,62 @@
 from fractal.common.models import TaskUpdate
 
 
-async def task_list(
-    client: AuthClient,
-    **kwargs,
-) -> RichJsonInterface:
-    res = await client.get(
-        f"{settings.BASE_URL}/task/",
-    )
+async def get_cached_task_by_name(name: str, client: AuthClient) -> int:
+    # Set paths
+    cache_dir = str(Path(f"{settings.FRACTAL_CACHE_PATH}").expanduser())
+    cache_file = f"{cache_dir}/tasks"
+
+    # If cache is missing, create it
+    cache_up_to_date = False
+    if not os.path.isfile(cache_file):
+        await refresh_task_cache(client)
+        cache_up_to_date = True
+
+    # Read cache
+    with open(cache_file, "r") as f:
+        task_cache = json.load(f)
```

Comment:
OK for now.
FYI, a bit redundant to go from Path to str to file handle.
cf. https://docs.python.org/3/library/pathlib.html?highlight=pathlib%20path#pathlib.Path.mkdir

Reply:
I streamlined it with https://github.com/fractal-analytics-platform/fractal/pull/273/commits/ba5688a7ba09d612de61f617980716acb32071a3